content/en/users/data/management/rucio/_index.md

```diff
@@ -26,6 +26,48 @@ Petabytes. The fact that the underlying Rucio infrastructure is managed by STFC,
 allows communities to easily start using and/or test Rucio with little setup
 cost.
 
+## Requirements to consider to use Rucio
+
+For Rucio to mangage your data in this setup, Rucio will need X509 certificate
+access, or soon, through
+[EGI Check-in](../../../../providers/check-in/_index.md) to:
+
+- Your experiments Storage Element that is X509 capable
+- Your experiments Tape Archive that is X509 capable
+- Your experiments Voms server information to check Users credentials against
+
+Rucio is a system that sits on top of already established storage elements to
+unify users access for data management, and retrival. Rucio consists of a
+database of the storage element's details, users and their access credential
+information, and access levels, and the data and its location, not a direct data
+storage solution.
+
+## Rucio Use Cases
+
+Rucio is a data management software, that integrates with your experiments
+currently provisioned storage. Therefore, this section will deal with usecases
+that you should consider Rucio for.
+
+### Management of data for archive when no longer used actively
+
+One that may be useful is if your experiment has data that is newly generated,
```

Comment:
not so much clear this paragraph

Reply:
rewrote in latest PR, hopefully makes more sense to you